logger.js

```diff
@@ -50,17 +51,19 @@ const Logger = (serviceName,
     if (err) {
       errorMessage = { error: { message: err.message,
                                 stack_trace: err.stack.split('\n') }}}
-    const totalMessage = { ...{ log: { level: severity },
-                                 '@timestamp': now(),
-                                 service: { name: serviceName }},
-                            ...defaultProperties,
-                            ...errorMessage,
-                            ...message }
+    const totalMessage = { ...{ ecs: { version: ECS_VERSION },
+                                log: { level: severity },
+                                '@timestamp': now(),
+                                service: { name: serviceName }},
+                           ...defaultProperties,
+                           ...errorMessage,
+                           ...message }
     const nestedMessage = jsonHelper(totalMessage)
     output.log(JSON.stringify(nestedMessage))
   }
 
   return {
+    ECS_VERSION: ECS_VERSION,
```

Comment:
Why the key is not in small case?

Reply:
it's not a (log) key, it's a symbolic constant.